scripts/repo-setup.sh

```diff
@@ -29,6 +29,18 @@ validateNginxKey() {
     fi
 }
 
+validatePgKey() {
+    pgFingerprint=$1
+    # source: https://wiki.postgresql.org/wiki/Apt
+    expectedPgFingerprint=B97B0AFCAA1A47F044F244A07FCC7D46ACCC4CF8
+
+    if [[ "$pgFingerprint" != "$expectedPgFingerprint" ]]; then
+        printf "ERROR: Invalid postgresql key fingerprint\n" >&2
+        apt-key del 7FCC7D46ACCC4CF8
```

Comment:
could you elaborate a bit here?

Reply:
We are validating that the Postgresql APT Repository key fingerprint, as published at https://www.postgresql.org/media/keys/ACCC4CF8.asc per https://wiki.postgresql.org/wiki/Apt, is what is expected. 

This, making sure that the Postgresql APT Repository that we are configuring in the customer's ubuntu system, is indeed that, and is not tampered with. 

If that is not the case, we print `ERROR: Invalid postgresql key fingerprint` , deleting the key from the system, and failing. 


Same is happening for Nginx Repository here: https://github.com/mattermost/mattermost-omnibus/blob/main/scripts/repo-setup.sh#L20-L30